uiowa/uiowa
# i3077 - Made featured image required for aos and added to config split

Closes #3077 


# How to test


1. blt ds --site=admissions.uiowa.edu
2. Go to http://admissions.local.drupal.uiowa.edu/admin/structure/types/manage/area_of_study/fields/node.area_of_study.field_image and make sure featured image is required in AOS content type
